tecken/symbolicate/views.py

```diff
@@ -165,7 +165,7 @@ def symbolicate(self, stacks, memory_map):
         needed_modules_lookups = {
             key: modules_lookups[key]
             for key in modules_lookups
-            if key not in self.all_symbol_offsets
+            if key[0] and key[1] and key not in self.all_symbol_offsets
```

Comment:
`key[0]` and `key[1]` can be the empty string. If they are, we don't want to try to look them up. We want to leave them as `None`.